exception/ExceptionHandler.js

```diff
@@ -115,4 +117,10 @@ export class ExceptionHandler {
     static sessionExpired(exception) {
         return exception && exception.httpStatus === 401;
     }
+
+    static cleanStack(exception) {
+        // statuses of 0, 4XX, 5XX are server errors, so the javascript stack
+        // is irrelevant and potentially misleading
+        if (/^[045]/.test(exception.httpStatus)) delete exception.stack;
```

Comment:
That's great - I had been noticing this and am glad we cleaned it up here.  (Assuming that in general we weren't getting useful info here in terms of what code path triggered the call to the server.)

Reply:
Correct.  The stack was starting with the line that threw the error as a response to the bad server response.